### PR TITLE
login functionality Test Automation

### PR DESCRIPTION
**Pull request by Gujalwar**
Local test are, ready for review
image
done, ready for review

### CODE REVIEW

**Comment by Gujalwar** [comment]
please review my PR

**Comment by Gujalwar** [comment]
looks good to me

**Comment by Ozleblebici** [comment]
if you can open PR again, I can review now @Gujalwar

**Comment by Gujalwar** [comment]
I did but not sure what I am doing. need more practice.

**Comment by Ozleblebici** [comment]
ok no worries, I opened again, pls see my reviews
also request review from your teammates on class chat

File: src/LoginFunctionalityTest.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+public class LoginFunctionalityTest {
+    public static void main(String[] args) {
+        System.out.println("navigate to log in page");
+        System.out.println("Enter username");
+        System.out.println("enter password");
+        System.out.println("verify login page");
```
Comment (Ozleblebici) [inline src/LoginFunctionalityTest.java:6]: please try to add forgotten password tests

Comment (Gujalwar) [reply src/LoginFunctionalityTest.java:6]: added

Comment (Ozleblebici) [reply src/LoginFunctionalityTest.java:6]: it should be reviewed and resolved by raised person  :) 
Look good

**Review by Ozleblebici — APPROVED** [review]
LGTM

**Pull request merged**